docs/deployment/teams.md

```diff
@@ -321,15 +321,15 @@ An access token is a secure credential that authorizes a user to interact with t
 
 Semgrep supports several types of access tokens:
 
-**User-generated (Web API) tokens**  
+#### User-generated (Web API) tokens 
 
 These tokens are created by admins in Semgrep AppSec Platform. They are used for API access, integrations, and automation. 
 
 Some features of these tokens:
 - Not tied to a specific user account and remain valid until manually revoked, even if the creator is no longer associated with the deployment.  
 - Only admins can generate or manage these tokens. This can be done by going to **Settings → Tokens** in Semgrep AppSec Platform.  
```

Comment:
```suggestion
- Only admins can generate or manage these tokens. Token management is found under **Settings → Tokens** in Semgrep AppSec Platform.  
```
This specific wording might  not be better - but I don't love "this can be done by". Also would be open to "You can manage tokens by"